src/realTimeReporter/ipc/client.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright 2020 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+import ipc from 'node-ipc';
+import { EVENTS } from '../../constants';
+
+export const connectIPCClient = () => {
+  ipc.config.id = `reportPortalReporter_${process.pid}`;
+  ipc.config.retry = 1500;
```

Comment:
Why exactly this value? `1500`

Reply:
This value is obtained experimentally. This time is enough to exclude the possibility of a client connecting to the server before it starts.